src/client/components/PropertyListItem/index.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { Card, Image, Button, CardContent, Grid, CardMeta, CardDescription, Container, Icon, Header, Label } from 'semantic-ui-react';
+import './index.scss';
+
+export default class PropertyListItem extends React.Component {
```

Comment:
Please, don't export the class declaration as default. Use `export class Proper...` instead.